eos-phone-home

```diff
@@ -410,7 +411,9 @@ class PhoneHome(object):
         return not os.path.exists(self._activated_path)
 
     def _do_activate(self):
-        if self._send_to_server(self.ACTIVATION_ENDPOINT,
+        log.debug("Sending activation data to", self._activation_endpoint)
```

Comment:
Needs `%s` at the end of the log message. Without this you'll get `TypeError: not all arguments converted during string formatting`. Ditto in `_do_ping`